src/components/BooksList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Books from './Books';
+
+const BooksList = () => {
+  const bookList = [{ id: 1, title: 'some', category: 'Action' }];
+  return (
+    <table>
+      <tr>
+        <td>id</td>
+        <td>title</td>
+        <td>category</td>
+      </tr>
+      {bookList.map(book => (
+        <Books
+          id={book.id}
+          key={book.id}
+          title={book.title}
+          category={book.category}
+        />
+      ))}
+    </table>
+  );
+};
```

Comment:
Welldone 👍 

Let's update the code.

You need to do the following;
1. Improve the file structure: `BookList` should be placed in `src/containers` directory.
2. `BookList` should connect to Redux store and fetches books from the state using [mapStateToProps](https://react-redux.js.org/using-react-redux/connect-mapstate) [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png)